packages/contracts/src/tokenDetails/logos/rbtc.ts

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/no-anonymous-default-export
+export default ``;
```

Comment:
Pls, ask Gilad for the logo.

Reply:
Created task for this, we'll leave it out of this PR for the moment - https://sovryn.atlassian.net/browse/SOV-1111